assets/vsphere.yaml

```diff
@@ -5,5 +5,6 @@ metadata:
   annotations:
     storageclass.kubernetes.io/is-default-class: "true"
 provisioner: kubernetes.io/vsphere-volume
+volumeBindingMode: WaitForFirstConsumer
```

Comment:
vsphere does not support topology aware dynamic provisioning. Should we verify nothing breaks if this option is enabled?